spec/features/index_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe 'local index page' do
   before do
-    visit "/Users/Typo32/Projects/my_project/lib/index.html"
+    visit '/Users/Typo32/Projects/my_project/lib/index.html'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.